library/dash/src/main/java/com/google/android/exoplayer2/source/dash/DashMediaSource.java

```diff
@@ -585,6 +586,16 @@ private void resolveUtcTimingElement(UtcTimingElement timingElement) {
     } else if (Util.areEqual(scheme, "urn:mpeg:dash:utc:http-xsdate:2014")
         || Util.areEqual(scheme, "urn:mpeg:dash:utc:http-xsdate:2012")) {
       resolveUtcTimingElementHttp(timingElement, new XsDateTimeParser());
+    } else if (Util.areEqual(scheme, "urn:mpeg:dash:utc:http-head:2012")
+        || Util.areEqual(scheme, "urn:mpeg:dash:utc:http-head:2014")) {
+      // According to US 9426196 B2 and ISO 23009-1, url could be a white-space
+      // separated list of URLs.
+      // As workaround for current architecture, just handle first one.
+      // TODO: find solution to support multiple urls, covered by US 9426196 B2 & ISO 23009-1
```

Comment:
Not really sure what this means, although perhaps once I find the right spec all will become clear :).